e2e/scheduler_test.go

```diff
@@ -33,21 +32,12 @@ func testScheduler() {
 		}
 
 		Eventually(func() error {
-			result, _, err := kubectl("get", "-n=topolvm-system", "pods", "--selector=app.kubernetes.io/component=scheduler,app.kubernetes.io/name=topolvm", "-o=json")
-			if err != nil {
-				return err
-			}
-
 			var podlist corev1.PodList
-			err = json.Unmarshal(result, &podlist)
+			err := getObjects(&podlist, "pods", "-n", "topolvm-system", "--selector=app.kubernetes.io/component=scheduler,app.kubernetes.io/name=topolvm")
 			if err != nil {
 				return err
 			}
 
-			if len(podlist.Items) == 0 {
```

Comment:
getObjects returns error if podlist is empty, so this is not needed.

Reply:
I use `--ignore-not-found` on getObjects. kubectl with the option seems to return an empty string rather than a json contains empty list.